src/transforms/transform-deployment.ts

```diff
@@ -48,30 +53,89 @@ const getLastSuccessfulDeployCommitSha = async (
 	return deployments[deployments.length - 1].sha;
 };
 
-const getCommitsSinceLastSuccessfulDeployment = async (
+const getLastSuccessDeploymentShaFromDyanmoDB = async (
```

Comment:
nit: I feel like "dynamodb" suffix is implementation detail, should we maybe drop it? `getLastPersisted(/Cached)SuccessfulDeployemtnSha` ?